tests/cases/fourslash/fourslash.ts

```diff
@@ -201,14 +201,6 @@ declare namespace FourSlashInterface {
         assertHasRanges(ranges: Range[]): void;
         caretAtMarker(markerName?: string): void;
         completionsAt(markerName: string | ReadonlyArray<string>, completions: ReadonlyArray<string | { name: string, insertText?: string, replacementSpan?: Range }>, options?: CompletionsAtOptions): void;
-        completionsAndDetailsAt(

```

Comment:
I removed this from the fourslash `.d.ts` because I tried to use it then discovered that it doesn't exist at runtime - the `verify` class doesn't have this member (any more?).